### ISSUE TITLE
Plugins installation fail

### ISSUE THREAD

**Issue by ikus060** [lektor/lektor#466]
Hello, 

On debian 9, I have a problem with the installlation of the plugins. I did not experience this error with debian 8. 
When executing lektor plugins reinstall:
```
Updating packages in /home/karlouche/.cache/lektor/packages/9ee5a59f1d889075d1e18e49272f57e8 for project
Collecting lektor-disqus-comments==0.2
  Using cached lektor_disqus_comments-0.2-py2-none-any.whl
Installing collected packages: lektor-disqus-comments
Exception:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python2.7/dist-packages/pip/commands/install.py", line 360, in run
    prefix=options.prefix_path,
  File "/usr/lib/python2.7/dist-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/usr/lib/python2.7/dist-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/usr/lib/python2.7/dist-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/usr/lib/python2.7/dist-packages/pip/wheel.py", line 247, in move_wheel_files
    prefix=prefix,
  File "/usr/lib/python2.7/dist-packages/pip/locations.py", line 153, in distutils_scheme
    i.finalize_options()
  File "/usr/lib/python2.7/distutils/command/install.py", line 289, in finalize_options
    raise DistutilsOptionError("can't combine user with prefix, "
DistutilsOptionError: can't combine user with prefix, exec_prefix/home, or install_(plat)base
Traceback (most recent call last):
  File "/usr/local/bin/lektor", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/lektor/cli.py", line 509, in plugins_list_cmd
    ctx.load_plugins()
  File "/usr/local/lib/python2.7/dist-packages/lektor/cli.py", line 96, in load_plugins
    load_packages(self.get_env(), reinstall=reinstall)
  File "/usr/local/lib/python2.7/dist-packages/lektor/packages.py", line 304, in load_packages
    refresh=reinstall)
  File "/usr/local/lib/python2.7/dist-packages/lektor/packages.py", line 274, in update_cache
    download_and_install_package(package_root, package, version)
  File "/usr/local/lib/python2.7/dist-packages/lektor/packages.py", line 103, in download_and_install_package
    raise RuntimeError('Failed to install dependency package.')
RuntimeError: Failed to install dependency package.
```

**Comment by karlb** [comment]
This bug is also tracked by debian https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=905147. Unfortunately, there's also no fix given in that ticket.

**Comment by nixjdm** [comment]
Thanks for bringing this back up. This should be closed now for the same reason as #596. The original reporter of that Debian ticket says this is fixed. Thanks!
